server/svix-server/src/worker.rs

```diff
@@ -62,11 +75,40 @@ pub struct FailureCacheValue {
     pub first_failure_at: DateTimeUtc,
 }
 
-kv_def!(FailureCacheKey, FailureCacheValue, "SVIX_FAILURE_CACHE");
+impl TryFrom<String> for FailureCacheValue {
+    type Error = Error;
+    fn try_from(s: String) -> error::Result<Self> {
+        Ok(FailureCacheValue {
+            first_failure_at: s.parse().map_err(|e| err_generic!(e))?,
+        })
+    }
+}
+
+impl std::fmt::Display for FailureCacheValue {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.first_failure_at)
+    }
+}
+
+impl crate::core::cache::CacheValue for FailureCacheValue {
```

Comment:
I don't know why the macro is not successfully handling this. Need to fix. :neutral_face: